python/pyspark/sql/session.py

```diff
@@ -599,6 +609,9 @@ def createDataFrame(self, data, schema=None, samplingRatio=None, verifySchema=Tr
             the sample ratio of rows used for inferring
         verifySchema : bool, optional
             verify data types of every row against schema. Enabled by default.
+            Specifically, if schema is provided, schema verification will be performed at the
+            preparation phase; if schema is not provided and can be inferred (eg. from UDT),
+            schema verification will be performed at the convertion phase.
```

Comment:
There is no such `preparation` phase exposed to users. I think it is confusing to mention the schema verification is performed at preparation phase. Could we remove it?

E.g. "Specifically, if schema is provided, schema verification will be performed against the provided schema; if schema is not provided and can be inferred (eg. from UDT), schema verification will be performed against the inferred schema."

Reply:
comment updated